pages/profile/[id]/index.js

```diff
@@ -60,94 +61,47 @@ function Profile() {
   return (
     <>
       <Header />
-      <div style={{ maxWidth: 1400, margin: "auto" }}>
-        <div className={profileStyles.profile}>
-          <div className={profileStyles.cover}>
-            <img src="/images/talentCover.png" />
-          </div>
-          <div className={profileStyles.bio}>
-            <div className={profileStyles.avatar}>
-              <img
-                alt="Identicon"
-                src={avatar}
-                loading="lazy"
-                className="sc-eirseW evgNzS"
-              />
+      <MainWrapper>
+        <ProfileContainer>
+          <div><img src="/images/talentCover.png" /></div>
```

Comment:
what is the reason for this div?
we should avoid to insert unnecceserly HTML element into DOM. for now its ok but keep in mind

Reply:
mainwrapper? 
the div that has max width of 1400px with 20px paddings, i placed that in global styled component which can be used anywhere needed.

Reply:
and if you mean the div which has img inside it, i removed that